frontend/src/app/testResults/TestResultCorrectionModal.tsx

```diff
@@ -70,39 +127,83 @@ export const DetachedTestResultCorrectionModal = ({
       });
   };
 
+  const markAsCorrection = () => {
+    markTestAsCorrection({
+      variables: {
+        id: testResultId,
+        reason,
+      },
+    })
+      .then(() => {
+        const alert = (
+          // TODO: better text here, maybe indicating to user that the test should now
+          // be available in the queue
+          <Alert type="success" title="Result marked as correction" body="" />
```

Comment:
Any suggestions on how to signal to the user that the test has been re-opened in the queue? We could mention it in the alert, or we could simply redirect to the queue (this would likely replace the alert)

Reply:
merging this, created this [pr](https://github.com/CDCgov/prime-simplereport/pull/3698) to address the automatic routing to queue